plugins/plugins.go

```diff
@@ -382,27 +385,15 @@ func New(raw []byte, id string, store storage.Store, opts ...func(*Manager)) (*M
 		return nil, err
 	}
 
-	keys, err := keys.ParseKeysConfig(parsedConfig.Keys)
-	if err != nil {
-		return nil, err
-	}
-
-	interQueryBuiltinCacheConfig, err := cache.ParseCachingConfig(parsedConfig.Caching)
-	if err != nil {
-		return nil, err
-	}
-
 	m := &Manager{
-		Store:                        store,
-		Config:                       parsedConfig,
-		ID:                           id,
-		keys:                         keys,
-		pluginStatus:                 map[string]*Status{},
-		pluginStatusListeners:        map[string]StatusListener{},
-		maxErrors:                    -1,
-		interQueryBuiltinCacheConfig: interQueryBuiltinCacheConfig,
-		serverInitialized:            make(chan struct{}),
-		bootstrapConfigLabels:        parsedConfig.Labels,
+		Store:                 store,
+		Config:                parsedConfig,
+		ID:                    id,
+		pluginStatus:          map[string]*Status{},
+		pluginStatusListeners: map[string]StatusListener{},
+		maxErrors:             -1,
+		serverInitialized:     make(chan struct{}),
+		bootstrapConfigLabels: parsedConfig.Labels,
```

Comment:
[comment]: This change makes sense in light of what happens a few lines further down in this function; we're delaying populating some of the Manager's fields until *after* all the overrides have happened.